### PR TITLE
Doischer/3162 implement eav for accounts

### PR DESCRIPTION
**Pull request by tomdoischer**
https://redmine.czechidm.com/issues/3162

### CODE REVIEW

**Comment by tomdoischer** [comment]
https://redmine.czechidm.com/issues/3162

File: Realization/backend/acc/src/main/java/eu/bcvsolutions/idm/acc/repository/AccSchemaFormAttributeRepository.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package eu.bcvsolutions.idm.acc.repository;
+
+import eu.bcvsolutions.idm.acc.entity.AccSchemaFormAttribute;
+import eu.bcvsolutions.idm.core.api.repository.AbstractEntityRepository;
+
+/**
+ * Extended attributes for accounts.
```
[the remark this answers is not in the document]
Comment (tomdoischer) [reply Realization/backend/acc/src/main/java/eu/bcvsolutions/idm/acc/repository/AccSchemaFormAttributeRepository.java:7]: I will slightly rewrite this, one repository is for values, and this one is for form attributes.

File: Realization/backend/acc/src/main/java/eu/bcvsolutions/idm/acc/rest/impl/AccAccountController.java [hunk 2]
```diff
@@ -332,6 +348,129 @@ public ResponseEntity<IcConnectorObject> getConnectorObject(
...
+		IdmFormDefinitionDto formDefinition = formDefinitionController.getDefinition(
+				AccAccount.class, 
+				definitionCode, 
+				IdmBasePermission.AUTOCOMPLETE);
+		//
+		return formDefinitionController.saveFormValues(dto, formDefinition, formValues, IdmBasePermission.UPDATE);
+	}
+	
 	@Override
 	protected AccAccountFilter toFilter(MultiValueMap<String, Object> parameters) {
```
[the remark this answers is not in the document]
Comment (tomdoischer) [reply Realization/backend/acc/src/main/java/eu/bcvsolutions/idm/acc/rest/impl/AccAccountController.java:473]: I had to rewrite it somewhat. I don't think that formdefinitionId should be used here (it's not this way in other controllers), form definition in toFilter shouldn't be necessary.

**Comment by kucera-roman** [comment]
LGTM, will be merged after https://github.com/bcvsolutions/CzechIdMng/pull/250

**Review by kucera-roman — APPROVED** [review]
LGTM, thx

**Review by kucera-roman — APPROVED** [review]
LGTM

**Pull request merged**